app/src/main/res/layout/activity_main.xml

```diff
@@ -47,28 +47,63 @@
         android:textStyle="bold"
         app:layout_constraintBottom_toTopOf="@+id/orderListPager"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.51"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/carousel" />
 
     <android.support.v4.view.ViewPager
-        android:foregroundGravity="center"
         android:id="@+id/orderListPager"
-        android:layout_centerInParent="true"
-        android:overScrollMode="never"
-        android:clipToPadding="false"
         android:layout_width="wrap_content"
         android:layout_height="100dp"
+        android:layout_centerInParent="true"
+        android:clipToPadding="false"
+        android:foregroundGravity="center"
+        android:overScrollMode="never"
         app:layout_constraintBottom_toTopOf="@id/button_checkout"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="1.0"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/text_order_header">
+
     </android.support.v4.view.ViewPager>
 
+    <Button
+        android:id="@+id/deleteItemButton"
+        android:layout_width="37dp"
```

Comment:
should create a constant in dimens.xml file, I will do it as part of the refactoring assignment